src/synth/fluid_voice.c

```diff
@@ -321,12 +321,13 @@ fluid_voice_init(fluid_voice_t *voice, fluid_sample_t *sample,
     voice->has_noteoff = 0;
     UPDATE_RVOICE0(fluid_rvoice_reset);
 
-    /* Increment the reference count of the sample to prevent the
-       unloading of the soundfont while this voice is playing,
-       once for us and once for the rvoice. */
+    /*
+       We increment the reference count of the sample to indicate that this
+       sample is about to be owned by the rvoice. This will prevent the
+       unloading of the soundfont while this rvoice is playing.
+    */
     fluid_sample_incr_ref(sample);
```

Comment:
Where is the matching `fluid_voice_sample_unref` for this one? You've removed one below and I can't find another one.

Reply:
Sry, scratch that. I got confused. Ofc `fluid_voice_sample_unref(&voice->rvoice->dsp.sample);` is the matching call.